diff -urN libzrtpcpp-1.0.0.org/src/Makefile.am libzrtpcpp-1.0.0/src/Makefile.am
--- libzrtpcpp-1.0.0.org/src/Makefile.am	2007-09-26 16:18:23.000000000 +0200
+++ libzrtpcpp-1.0.0/src/Makefile.am	2008-01-16 08:39:43.783400511 +0100
@@ -5,8 +5,6 @@
 lib_LTLIBRARIES = libzrtpcpp.la
 # libzrtpcpp_la_LIBADD = @LIBS@
 
-libzrtpcpp_la_LDFLAGS = $(RELEASE)
-
 # http://www.gnu.org/software/libtool/manual.html
 
 if ZRTP_OPENSSL
diff -urN libzrtpcpp-1.3.0.org/configure.ac libzrtpcpp-1.3.0/configure.ac
--- libzrtpcpp-1.3.0.org/configure.ac	2008-06-14 15:34:15.000000000 +0200
+++ libzrtpcpp-1.3.0/configure.ac	2008-08-23 13:07:25.837343793 +0200
@@ -28,8 +28,8 @@
 # AC_INIT([libzrtpcpp], [1.0.0] )
 # AC_CONFIG_SRCDIR([configure.ac])
 
-LT_RELEASE="1.4"
-LT_VERSION="0:3"
+TW_LT_RELEASE="1.4"
+TW_LT_VERSION="0:3"
 
 REQUIRES="1.3.0"
 
@@ -129,7 +129,7 @@
 
 OST_PROG_COMMON
 OST_PROG_LIBVER
-OST_PROG_LIBRARY(CCXX,[$LT_VERSION])
+OST_PROG_LIBRARY(CCXX,[$TW_LT_VERSION])
 OST_AUTOMAKE_MODE
 OST_MAINTAINER_MODE
 # OST_CC_ENDIAN - now in Common C++
@@ -178,8 +178,8 @@
 AC_SUBST(DOXYGEN)
 AM_CONDITIONAL(DOXY, test "$DOXYGEN" != "no")
 
-LIB_VERSION=`echo $LT_RELEASE | sed -e 's/[\.]/_/'` 
-LIB_MAJOR=`echo $LT_VERSION | sed -e 's/[:].*$//'`
+LIB_VERSION=`echo $TW_LT_RELEASE | sed -e 's/[\.]/_/'` 
+LIB_MAJOR=`echo $TW_LT_VERSION | sed -e 's/[:].*$//'`
 
 AC_SUBST(LIB_VERSION)
 AC_SUBST(LIB_MAJOR)
diff -urN libzrtpcpp-1.3.0.org/demo/Makefile.am libzrtpcpp-1.3.0/demo/Makefile.am
--- libzrtpcpp-1.3.0.org/demo/Makefile.am	2008-04-13 15:00:50.000000000 +0200
+++ libzrtpcpp-1.3.0/demo/Makefile.am	2008-08-23 13:07:28.027759162 +0200
@@ -9,9 +9,9 @@
 # implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 
 AUTOMAKE_OPTIONS = no-dependencies dist-shar
-LT_VERSION=@LT_CCXX_VERSION@
+TW_LT_VERSION=@LT_CCXX_VERSION@
 #ignored for programs
-#$(LT_VERSION) -release $(LT_RELEASE)
+#$(TW_LT_VERSION) -release $(TW_LT_RELEASE)
 AM_CXXFLAGS = @WARN_FLAGS@ -I$(top_srcdir)/src
 # ccxxincludedir=$(includedir)/cc++
 
diff -urN libzrtpcpp-1.3.0.org/m4/ost_prog.m4 libzrtpcpp-1.3.0/m4/ost_prog.m4
--- libzrtpcpp-1.3.0.org/m4/ost_prog.m4	2006-09-10 20:40:24.000000000 +0200
+++ libzrtpcpp-1.3.0/m4/ost_prog.m4	2008-08-23 13:07:28.047345431 +0200
@@ -45,10 +45,10 @@
 ])
 
 AC_DEFUN([OST_PROG_LIBVER],[
-	LT_MAJOR="`echo $LT_VERSION | sed s/:.*$//`"
-	LT_MINOR="`echo $LT_VERSION | sed s/.*://`"
+	LT_MAJOR="`echo $TW_LT_VERSION | sed s/:.*$//`"
+	LT_MINOR="`echo $TW_LT_VERSION | sed s/.*://`"
 	LT_SUBVER="$LT_MAJOR.$LT_MINOR"
-	AC_SUBST(LT_RELEASE)
+	AC_SUBST(TW_LT_RELEASE)
 	AC_SUBST(LT_SUBVER)
 	AC_SUBST(LT_MAJOR)
 	AC_SUBST(LT_MINOR)
diff -urN libzrtpcpp-1.3.0.org/src/Makefile.am libzrtpcpp-1.3.0/src/Makefile.am
--- libzrtpcpp-1.3.0.org/src/Makefile.am	2008-03-16 12:53:33.000000000 +0100
+++ libzrtpcpp-1.3.0/src/Makefile.am	2008-08-23 13:07:28.050676134 +0200
@@ -1,8 +1,8 @@
 # Copyright (C) 2006 Werner Dittmann.
 
-LT_VERSION=@LT_CCXX_VERSION@
-RELEASE = $(LT_VERSION) -release $(LT_RELEASE)
-#RELEASE = $(LT_VERSION)
+TW_LT_VERSION=@LT_CCXX_VERSION@
+RELEASE = $(TW_LT_VERSION) -release $(TW_LT_RELEASE)
+#RELEASE = $(TW_LT_VERSION)
 lib_LTLIBRARIES = libzrtpcpp.la
 # libzrtpcpp_la_LIBADD = @LIBS@
 

