diff -urN libzrtpcpp-1.0.0.org/src/Makefile.am libzrtpcpp-1.0.0/src/Makefile.am
--- libzrtpcpp-1.0.0.org/src/Makefile.am	2007-09-26 16:18:23.000000000 +0200
+++ libzrtpcpp-1.0.0/src/Makefile.am	2008-01-16 08:39:43.783400511 +0100
@@ -5,8 +5,6 @@
 lib_LTLIBRARIES = libzrtpcpp.la
 # libzrtpcpp_la_LIBADD = @LIBS@
 
-libzrtpcpp_la_LDFLAGS = $(RELEASE)
-
 # http://www.gnu.org/software/libtool/manual.html
 
 if ZRTP_OPENSSL
diff -urN libzrtpcpp-1.3.0.org/configure.ac libzrtpcpp-1.3.0/configure.ac
--- libzrtpcpp-1.3.0.org/configure.ac	2008-06-14 15:34:15.000000000 +0200
+++ libzrtpcpp-1.3.0/configure.ac	2008-08-23 13:04:33.287360894 +0200
@@ -173,7 +173,9 @@
 AC_SUBST(DOXYGEN)
 AM_CONDITIONAL(DOXY, test "$DOXYGEN" != "no")
 
-AC_SUBST(CRYPTOBACKEND PACKAGE_REQ BUILD_REQ)
+AC_SUBST(CRYPTOBACKEND)
+AC_SUBST(PACKAGE_REQ)
+AC_SUBST(BUILD_REQ)
 AC_CONFIG_FILES([Makefile src/Makefile src/libzrtpcpp/Makefile m4/Makefile
                 src/libzrtpcpp/crypto/Makefile doc/Makefile demo/Makefile
                 libzrtpcpp.pc libzrtpcpp.spec])  # m4/Makefile
diff -urN libzrtpcpp-1.3.0.org/configure.ac libzrtpcpp-1.3.0/configure.ac
--- libzrtpcpp-1.3.0.org/configure.ac	2008-06-14 15:34:15.000000000 +0200
+++ libzrtpcpp-1.3.0/configure.ac	2008-08-23 13:07:25.837343793 +0200
@@ -10,26 +10,26 @@
 #
 AC_INIT(src/ZRtp.cxx)
 
-# LT_VERSION actually is meant to be in the form ABI-LINK_VER:ABI_PATCH_VER. 
+# TW_LT_VERSION actually is meant to be in the form ABI-LINK_VER:ABI_PATCH_VER. 
 # This eventaully goes into libtool as "version-info" (via LT_CCXX_VERSION in
-# Makefile.am). LT_RELEASE goes into libtool as "release".
+# Makefile.am). TW_LT_RELEASE goes into libtool as "release".
 #
 # Hence, if the binary api is not changed for a given release, 1.0.0 should have:
 #
 # VERSION=1.0.0
-# LT_RELEASE=1.0
-# LT_VERSION=0:0
+# TW_LT_RELEASE=1.0
+# TW_LT_VERSION=0:0
 #
-# VERSION=1.0.1 might have either LT_VERSION=0:1 if binary linking remains 
-# unchanged, or LT_VERSION=1:0 if binary linking is changed.
+# VERSION=1.0.1 might have either TW_LT_VERSION=0:1 if binary linking remains 
+# unchanged, or TW_LT_VERSION=1:0 if binary linking is changed.
 # (see macros "OST_PROG_LIBVER" and "OST_PROG_LIBRARY" below)
 #
 VERSION="1.3.0"
 # AC_INIT([libzrtpcpp], [1.0.0] )
 # AC_CONFIG_SRCDIR([configure.ac])
 
-LT_RELEASE="1.3"
-LT_VERSION="0:0"
+TW_LT_RELEASE="1.3"
+TW_LT_VERSION="0:0"
 
 REQUIRES="1.3.0"
 
@@ -125,7 +125,7 @@
 
 OST_PROG_COMMON
 OST_PROG_LIBVER
-OST_PROG_LIBRARY(CCXX,[$LT_VERSION])
+OST_PROG_LIBRARY(CCXX,[$TW_LT_VERSION])
 OST_AUTOMAKE_MODE
 OST_MAINTAINER_MODE
 # OST_CC_ENDIAN - now in Common C++
diff -urN libzrtpcpp-1.3.0.org/demo/Makefile.am libzrtpcpp-1.3.0/demo/Makefile.am
--- libzrtpcpp-1.3.0.org/demo/Makefile.am	2008-04-13 15:00:50.000000000 +0200
+++ libzrtpcpp-1.3.0/demo/Makefile.am	2008-08-23 13:07:28.027759162 +0200
@@ -9,9 +9,9 @@
 # implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 
 AUTOMAKE_OPTIONS = no-dependencies dist-shar
-LT_VERSION=@LT_CCXX_VERSION@
+TW_LT_VERSION=@LT_CCXX_VERSION@
 #ignored for programs
-#$(LT_VERSION) -release $(LT_RELEASE)
+#$(TW_LT_VERSION) -release $(TW_LT_RELEASE)
 AM_CXXFLAGS = @WARN_FLAGS@ -I$(top_srcdir)/src
 # ccxxincludedir=$(includedir)/cc++
 
diff -urN libzrtpcpp-1.3.0.org/demo/Makefile.in libzrtpcpp-1.3.0/demo/Makefile.in
--- libzrtpcpp-1.3.0.org/demo/Makefile.in	2008-06-14 15:37:27.000000000 +0200
+++ libzrtpcpp-1.3.0/demo/Makefile.in	2008-08-23 13:07:28.027759162 +0200
@@ -129,7 +129,7 @@
 LT_CCXX_VERSION = @LT_CCXX_VERSION@
 LT_MAJOR = @LT_MAJOR@
 LT_MINOR = @LT_MINOR@
-LT_RELEASE = @LT_RELEASE@
+TW_LT_RELEASE = @TW_LT_RELEASE@
 LT_SUBVER = @LT_SUBVER@
 MAINT = @MAINT@
 MAKEINFO = @MAKEINFO@
@@ -209,9 +209,9 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AUTOMAKE_OPTIONS = no-dependencies dist-shar
-LT_VERSION = @LT_CCXX_VERSION@
+TW_LT_VERSION = @LT_CCXX_VERSION@
 #ignored for programs
-#$(LT_VERSION) -release $(LT_RELEASE)
+#$(TW_LT_VERSION) -release $(TW_LT_RELEASE)
 AM_CXXFLAGS = @WARN_FLAGS@ -I$(top_srcdir)/src
 zrtptest_SOURCES = zrtptest.cpp
 zrtptest_LDFLAGS = ../src/libzrtpcpp.la
diff -urN libzrtpcpp-1.3.0.org/doc/Makefile.in libzrtpcpp-1.3.0/doc/Makefile.in
--- libzrtpcpp-1.3.0.org/doc/Makefile.in	2008-06-14 15:37:27.000000000 +0200
+++ libzrtpcpp-1.3.0/doc/Makefile.in	2008-08-23 13:07:28.034425891 +0200
@@ -110,7 +110,7 @@
 LT_CCXX_VERSION = @LT_CCXX_VERSION@
 LT_MAJOR = @LT_MAJOR@
 LT_MINOR = @LT_MINOR@
-LT_RELEASE = @LT_RELEASE@
+TW_LT_RELEASE = @TW_LT_RELEASE@
 LT_SUBVER = @LT_SUBVER@
 MAINT = @MAINT@
 MAKEINFO = @MAKEINFO@
diff -urN libzrtpcpp-1.3.0.org/m4/Makefile.in libzrtpcpp-1.3.0/m4/Makefile.in
--- libzrtpcpp-1.3.0.org/m4/Makefile.in	2008-06-14 15:37:27.000000000 +0200
+++ libzrtpcpp-1.3.0/m4/Makefile.in	2008-08-23 13:07:28.047345431 +0200
@@ -107,7 +107,7 @@
 LT_CCXX_VERSION = @LT_CCXX_VERSION@
 LT_MAJOR = @LT_MAJOR@
 LT_MINOR = @LT_MINOR@
-LT_RELEASE = @LT_RELEASE@
+TW_LT_RELEASE = @TW_LT_RELEASE@
 LT_SUBVER = @LT_SUBVER@
 MAINT = @MAINT@
 MAKEINFO = @MAKEINFO@
diff -urN libzrtpcpp-1.3.0.org/m4/ost_prog.m4 libzrtpcpp-1.3.0/m4/ost_prog.m4
--- libzrtpcpp-1.3.0.org/m4/ost_prog.m4	2006-09-10 20:40:24.000000000 +0200
+++ libzrtpcpp-1.3.0/m4/ost_prog.m4	2008-08-23 13:07:28.047345431 +0200
@@ -45,10 +45,10 @@
 ])
 
 AC_DEFUN([OST_PROG_LIBVER],[
-	LT_MAJOR="`echo $LT_VERSION | sed s/:.*$//`"
-	LT_MINOR="`echo $LT_VERSION | sed s/.*://`"
+	LT_MAJOR="`echo $TW_LT_VERSION | sed s/:.*$//`"
+	LT_MINOR="`echo $TW_LT_VERSION | sed s/.*://`"
 	LT_SUBVER="$LT_MAJOR.$LT_MINOR"
-	AC_SUBST(LT_RELEASE)
+	AC_SUBST(TW_LT_RELEASE)
 	AC_SUBST(LT_SUBVER)
 	AC_SUBST(LT_MAJOR)
 	AC_SUBST(LT_MINOR)
diff -urN libzrtpcpp-1.3.0.org/Makefile.in libzrtpcpp-1.3.0/Makefile.in
--- libzrtpcpp-1.3.0.org/Makefile.in	2008-06-14 15:37:27.000000000 +0200
+++ libzrtpcpp-1.3.0/Makefile.in	2008-08-23 13:07:25.850676381 +0200
@@ -144,7 +144,7 @@
 LT_CCXX_VERSION = @LT_CCXX_VERSION@
 LT_MAJOR = @LT_MAJOR@
 LT_MINOR = @LT_MINOR@
-LT_RELEASE = @LT_RELEASE@
+TW_LT_RELEASE = @TW_LT_RELEASE@
 LT_SUBVER = @LT_SUBVER@
 MAINT = @MAINT@
 MAKEINFO = @MAKEINFO@
diff -urN libzrtpcpp-1.3.0.org/src/Makefile.am libzrtpcpp-1.3.0/src/Makefile.am
--- libzrtpcpp-1.3.0.org/src/Makefile.am	2008-03-16 12:53:33.000000000 +0100
+++ libzrtpcpp-1.3.0/src/Makefile.am	2008-08-23 13:07:28.050676134 +0200
@@ -1,8 +1,8 @@
 # Copyright (C) 2006 Werner Dittmann.
 
-LT_VERSION=@LT_CCXX_VERSION@
-RELEASE = $(LT_VERSION) -release $(LT_RELEASE)
-#RELEASE = $(LT_VERSION)
+TW_LT_VERSION=@LT_CCXX_VERSION@
+RELEASE = $(TW_LT_VERSION) -release $(TW_LT_RELEASE)
+#RELEASE = $(TW_LT_VERSION)
 lib_LTLIBRARIES = libzrtpcpp.la
 # libzrtpcpp_la_LIBADD = @LIBS@
 
diff -urN libzrtpcpp-1.3.0.org/src/Makefile.in libzrtpcpp-1.3.0/src/Makefile.in
--- libzrtpcpp-1.3.0.org/src/Makefile.in	2008-06-14 15:37:27.000000000 +0200
+++ libzrtpcpp-1.3.0/src/Makefile.in	2008-08-23 13:07:28.054009288 +0200
@@ -167,7 +167,7 @@
 LT_CCXX_VERSION = @LT_CCXX_VERSION@
 LT_MAJOR = @LT_MAJOR@
 LT_MINOR = @LT_MINOR@
-LT_RELEASE = @LT_RELEASE@
+TW_LT_RELEASE = @TW_LT_RELEASE@
 LT_SUBVER = @LT_SUBVER@
 MAINT = @MAINT@
 MAKEINFO = @MAKEINFO@
@@ -246,9 +246,9 @@
 target_vendor = @target_vendor@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-LT_VERSION = @LT_CCXX_VERSION@
-RELEASE = $(LT_VERSION) -release $(LT_RELEASE)
-#RELEASE = $(LT_VERSION)
+TW_LT_VERSION = @LT_CCXX_VERSION@
+RELEASE = $(TW_LT_VERSION) -release $(TW_LT_RELEASE)
+#RELEASE = $(TW_LT_VERSION)
 lib_LTLIBRARIES = libzrtpcpp.la
 # libzrtpcpp_la_LIBADD = @LIBS@
 libzrtpcpp_la_LDFLAGS = $(RELEASE)
